src/main/resources/assets/alexscaves/lang/zh_cn.json

```diff
@@ -52,124 +52,124 @@
-  "block.alexscaves.cave_painting_enderman.desc": "“末影人”",
-  "block.alexscaves.cave_painting_portal.desc": "“传送门”",
-  "block.alexscaves.cave_painting_subterranodon.desc": "“地底翼龙”",
-  "block.alexscaves.cave_painting_subterranodon_ride.desc": "“地底翼龙坐骑”",
-  "block.alexscaves.cave_painting_tremorsaurus.desc": "“撼地龙”",
-  "block.alexscaves.cave_painting_tremorsaurus_friend.desc": "“撼地龙伙伴”",
-  "block.alexscaves.cave_painting_mystery_1.desc": "“神秘壁画 I”",
-  "block.alexscaves.cave_painting_mystery_2.desc": "“神秘壁画 II”",
-  "block.alexscaves.cave_painting_mystery_3.desc": "“神秘壁画 III”",
-  "block.alexscaves.cave_painting_mystery_4.desc": "“神秘壁画 IV”",
-  "block.alexscaves.cave_painting_mystery_5.desc": "“神秘壁画 V”",
-  "block.alexscaves.cave_painting_mystery_6.desc": "“神秘壁画 VI”",
-  "block.alexscaves.cave_painting_mystery_7.desc": "“神秘壁画 VII”",
-  "block.alexscaves.cave_painting_mystery_8.desc": "“神秘壁画 VIII”",
-  "block.alexscaves.cave_painting_mystery_9.desc": "“神秘壁画 IX”",
+  "block.alexscaves.cave_painting_ambersol.desc": "\"日光琥珀\"",
+  "block.alexscaves.cave_painting_dark.desc": "\"黑暗\"",
+  "block.alexscaves.cave_painting_footprint.desc": "\"大型足迹\"",
+  "block.alexscaves.cave_painting_footprints.desc": "\"足迹\"",
+  "block.alexscaves.cave_painting_tree_stars.desc": "\"星形树叶\"",
+  "block.alexscaves.cave_painting_pewen.desc": "\"原始杉\"",
+  "block.alexscaves.cave_painting_trilocaris.desc": "\"三叶虾\"",
+  "block.alexscaves.cave_painting_grottoceratops.desc": "\"石窟角龙\"",
+  "block.alexscaves.cave_painting_grottoceratops_friend.desc": "\"与石窟角龙成为朋友\"",
+  "block.alexscaves.cave_painting_dino_nuggets.desc": "\"龙肉块\"",
+  "block.alexscaves.cave_painting_vallumraptor_chest.desc": "\"垒盗龙与宝箱\"",
+  "block.alexscaves.cave_painting_vallumraptor_friend.desc": "\"与垒盗龙成为朋友\"",
+  "block.alexscaves.cave_painting_relicheirus.desc": "\"祷手龙\"",
+  "block.alexscaves.cave_painting_relicheirus_slash.desc": "\"祷手龙与树\"",
+  "block.alexscaves.cave_painting_enderman.desc": "\"末影人\"",
+  "block.alexscaves.cave_painting_portal.desc": "\"传送门\"",
+  "block.alexscaves.cave_painting_subterranodon.desc": "\"地翼龙\"",
+  "block.alexscaves.cave_painting_subterranodon_ride.desc": "\"骑乘地翼龙\"",
+  "block.alexscaves.cave_painting_tremorsaurus.desc": "\"颤暴龙\"",
+  "block.alexscaves.cave_painting_tremorsaurus_friend.desc": "\"与颤暴龙成为朋友\"",
+  "block.alexscaves.cave_painting_mystery_1.desc": "\"神秘图案 I\"",
+  "block.alexscaves.cave_painting_mystery_2.desc": "\"神秘图案 II\"",
+  "block.alexscaves.cave_painting_mystery_3.desc": "\"神秘图案 III\"",
+  "block.alexscaves.cave_painting_mystery_4.desc": "\"神秘图案 IV\"",
+  "block.alexscaves.cave_painting_mystery_5.desc": "\"神秘图案 V\"",
+  "block.alexscaves.cave_painting_mystery_6.desc": "\"神秘图案 VI\"",
+  "block.alexscaves.cave_painting_mystery_7.desc": "\"神秘图案 VII\"",
+  "block.alexscaves.cave_painting_mystery_8.desc": "\"神秘图案 VIII\"",
+  "block.alexscaves.cave_painting_mystery_9.desc": "\"神秘图案 IX\"",

```

Comment:
**Improper quote marks.** These quote marks should use Chinese marks `“”` instead of English marks `""`.